package.json

```diff
@@ -61,15 +61,15 @@
     "node-forge": "^0.7.6"
```

Comment:
@hiranya911 Given that you are already updating the dependencies, could you update this one to 0.9.1. Given that it is a 0.x version, npm considers it to be a breaking change, and hence not compatible with the current requirement.

`node-forge` is also transitively being used by firestore (direct dependency is google-p12-pem), and requiring version ^0.9.0, which means two copies of the library are currently needed. Switching to just one dependency would save 1.7Mib.